src/index.html

```diff
@@ -1,18 +1,51 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
+    <meta charset="utf-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="../src/main.scss"
```

Comment:
The `href` attribute is pointing to a SCSS file (`../src/main.scss`). Browsers cannot directly interpret SCSS files. You need to compile the SCSS into a CSS file and link to that CSS file instead.